recipe/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "skbase" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sktime/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 085a97b1a5acb649a836b88ee5c30c8b9a8258baaee86c6ac73d366c2bae480f
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+    - toml 0.10.2
+    - build 0.10.0
+  run:
+    - python
+
+test:
+  source_files:
+    - skbase/tests
+  imports:
+    - skbase
+  commands:
+    - pip check
+    - pytest skbase/tests
+  requires:
+    - pip
+    - pytest
+    - numpy
+    - scipy
+    - pandas
```

```suggestion
    - pandas
    - scikit-learn >=0.24.0
```
https://github.com/sktime/skbase/blob/v0.6.0/pyproject.toml#L89C6-L89C26

---

I did not see `scikit-learn` being used anywhere.

---

not in python code at least.